packages/consumption/src/modules/attributes/AttributesController.ts

```diff
@@ -276,6 +278,20 @@ export class AttributesController extends ConsumptionBaseController {
         }
     }
 
+    private async setAsDefaultAttributeIfNoOtherExists(attribute: LocalAttribute): Promise<LocalAttribute> {
```

Comment:
I find the name a little ambiguous: if no other _what_ exists? Default attribute for the value type or attribute of same value type?

Reply:
suggestion: `setAsDefaultAttributeIfTypeUnique`

Reply:
Personally, I prefer `setAsDefaultAttributeIfNoOtherExists` over `setAsDefaultAttributeIfTypeUnique` since it is easier language. @slavistan Any opinion?

Reply:
>We always have a default attribute for each value type

Right. Keep the name then or use the other one, I don't mind either. But I do have a different suggestion: Have you thought about merging `changeDefaultRepositoryAttribute` and `setAsDefault..........` into a single method? They do similar things and could be implemented with a single `setAsDefaultRepositoryAttribute(attribute: LocalAttribute, override?: boolean)` where `override` would be a (false-if-undefined) boolean used to replace an existing default.

Reply:
In principle, that is a good idea, since as you pointed out they do similar things. However, I worry that having this as a public function then, somebody might mistakenly forget to set the overwrite flag and we'd end up with multiple default Attributes. Thus, I think it might be safer to keep them as two separate functions.

Reply:
If you want to override defaults then change the second parameter to `skipOverride?: boolean`. Does that solve the problem?